gneiss/regression/_format.py

```diff
@@ -0,0 +1,31 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, gneiss development team.
+#
+# Distributed under the terms of the GPLv3 License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+import qiime2.plugin.model as model
+from gneiss.plugin_setup import plugin
+import pickle
+
+
+class RegressionFormat_g(model.BinaryFileFormat):
+    def sniff(self):
+        try:
```

Comment:
you also might want to check the type of the pickled object to have a little bit of contents check